eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -95,34 +125,56 @@ private void refreshCurrent(IWorkbenchPage[] pages, IProject project, String fil
         }
     }
 
+    /**
+     * The refresher for Custom ReportList View. If secondary id is empty, 
+     * it checks if a refresh really needs to happen and if so updates every 
+     * custom view for the current project. If secondary-id is not empty, 
+     * it will search for the gives secondary-id custom view and 
+     * updates that particular one.
+     *
+     * @param pages the page list for the currently active workbench windows.
```

Comment:
-> currently active workbench window.
there can be only one active window, right?